build-test.cmd

```diff
@@ -360,6 +366,20 @@ if errorlevel 1 (
     exit /b 1
 )
 
+xcopy /s /y "%CORE_ROOT_STAGE%" "%CORE_ROOT%"
```

Comment:
It is, but when we build for non-Windows, we restore non-Windows binaries only, meaning we don't get clrjit.dll, etc., but rather clrjit.so. I believe there's other such dependencies as well (System.runtime, etc). I could get those by restoring a Core_Root for Windows & using that to Crossgen the non-windows Core_Root, but it's a bit of a hack